.github/workflows/unittest.yml

```diff
@@ -0,0 +1,42 @@
+name: Helm unit tests
+
+on:
+  push:
+    branches:
+      - main
+      - v[0-9]+
+      - v[0-9]+.[0-9]+
+      - cryostat-v[0-9]+.[0-9]+
+  
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
+      - unlabeled
+    branches:
+      - main
+      - v[0-9]+
+      - v[0-9]+.[0-9]+
+      - cryostat-v[0-9]+.[0-9]+
+
+jobs:
+  helm-unittests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: fail if safe-to-test label NOT applied
+        if: ${{ github.event_name == 'pull_request' && !contains(github.event.pull_request.labels.*.name, 'safe-to-test') }}
+        run: exit 1
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v4
+        with:
+          version: v3.15.2
```

Comment:
We are using `v3.14.4` in other CI. Let's keep it at `3.14` for now and bump later.

Reply:
or may be change `v3.14.4` to match the latest?

Reply:
Sure that would work! Though, it's better to bump the version in a separate PR.

Reply:
> If something goes wrong with fetching the latest version the action will use the hardcoded default stable version (currently v3.13.3). If you rely on a certain version higher than the default, you should explicitly use that version instead of latest.


We need to be careful there since helm `3.13` might be used, while the min helm version required is `3.14`.